sdk/eventgrid/azure-resourcemanager-eventgrid/src/main/java/com/azure/resourcemanager/eventgrid/implementation/Utils.java

```diff
@@ -84,7 +85,7 @@ static String getValueFromIdByParameterName(String id, String pathTemplate, Stri
         private final Function<PagedResponse<T>, PagedResponse<S>> pageMapper;
 
         private PagedIterableImpl(PagedIterable<T> pageIterable, Function<T, S> mapper) {
-            super(new PagedFlux<S>(Mono::empty));
+            super(new PagedFlux<S>(() -> Mono.empty()));
```

Comment:
The generated method reference apparently causes a compilation failure with Java 8 as `Mono` has a public method `empty()` and an internal method `empty(Publisher<T>)` which leads to an ambiguous call.

https://github.com/reactor/reactor-core/blame/master/reactor-core/src/main/java/reactor/core/publisher/Mono.java#L4905

cc: @srnagar @weidongxu-microsoft